docs/source/technical_features.md

```diff
@@ -32,15 +17,15 @@ positive and negative samples, negative samples being under-weighted dynamically
 at each epoch. During the second step, the CNN is fine-tuned on the realistic
 (i.e. class-imbalanced) dataset.
 
-### Mixup
+## Mixup
 [Mixup](https://arxiv.org/pdf/1710.09412.pdf) is a data augmentation technique,
```

Comment:
I could site the abstract link and not the pdf: https://arxiv.org/abs/1710.09412

Reply:
fixed in 93868ce7ffeba5a35b420e280f2d71716e1376f4